fpdf/template.py

```diff
@@ -74,26 +73,42 @@ def load_elements(self, elements):
         self.elements = elements
         self.keys = [v["name"].lower() for v in self.elements]
 
+    @staticmethod
+    def _parse_colorcode(s):
+        """ Allow hex and oct values for colors """
+        s = s.strip()
+        if not s:
+            raise ValueError('Foreground and Background must be numeric')
+        if s[:2] in ['0x', '0X']:
+            return int(s, 16)
+        if s[0] == '0':
+            return int(s, 8)
+        return int(s)
+
     def parse_csv(self, infile, delimiter=",", decimal_sep=".", encoding=None):
         """Parse template format csv file and create elements dict"""
-        keys = (
-            "name",
-            "type",
-            "x1",
-            "y1",
-            "x2",
-            "y2",
-            "font",
-            "size",
-            "bold",
-            "italic",
-            "underline",
-            "foreground",
-            "background",
-            "align",
-            "text",
-            "priority",
-            "multiline",
+        def varsep_float(s):
+            """Convert to float with given decimal seperator"""
+            # glad to have nonlocal scoping...
+            return float(s.replace(decimal_sep, '.'))
```

Comment:
I hadn't payed enough attention to decimal seperator replacement at first.
It previously was done on any field where the content didn't start with a ' (single quote).
Those are not at all garanteed to be present in text fields, so we need to restrict replacement to actual float value fields.
Fortunately a nested function sees the surrounding scope where the seperator is defined, which makes this easy.